components/chainhook-sdk/src/indexer/stacks/mod.rs

```diff
@@ -562,6 +591,198 @@ pub fn standardize_stacks_microblock_trail(
     Ok(microblocks)
 }
 
+#[cfg(feature = "stacks-signers")]
+pub fn standardize_stacks_marshalled_stackerdb_chunks(
+    indexer_config: &IndexerConfig,
+    marshalled_stackerdb_chunks: JsonValue,
+    chain_ctx: &mut StacksChainContext,
+    ctx: &Context,
+) -> Result<Vec<StacksStackerDbChunk>, String> {
+    let mut stackerdb_chunks: NewStackerDbChunks =
+        serde_json::from_value(marshalled_stackerdb_chunks)
+            .map_err(|e| format!("unable to parse stackerdb chunks {e}"))?;
+    standardize_stacks_stackerdb_chunks(indexer_config, &mut stackerdb_chunks, chain_ctx, ctx)
+}
+
+#[cfg(feature = "stacks-signers")]
+pub fn standardize_stacks_stackerdb_chunks(
+    indexer_config: &IndexerConfig,
+    stackerdb_chunks: &mut NewStackerDbChunks,
+    chain_ctx: &mut StacksChainContext,
+    ctx: &Context,
+) -> Result<Vec<StacksStackerDbChunk>, String> {
+    use stacks_codec::codec::BlockResponse;
+    use stacks_codec::codec::RejectCode;
+    use stacks_codec::codec::SignerMessage;
+    use stacks_codec::codec::ValidateRejectCode;
+
+    let contract_id = &stackerdb_chunks.contract_id.name;
+    let mut parsed_chunks: Vec<StacksStackerDbChunk> = vec![];
+    for slot in stackerdb_chunks.modified_slots.iter() {
+        let data_bytes = match hex::decode(&slot.data) {
+            Ok(bytes) => bytes,
+            Err(e) => return Err(format!("unable to decode signer slot hex data: {e}")),
+        };
+        let signer_message =
+            match SignerMessage::consensus_deserialize(&mut Cursor::new(&data_bytes)) {
+                Ok(message) => message,
+                Err(e) => return Err(format!("unable to deserialize SignerMessage: {e}")),
+            };
+        let message = match signer_message {
+            SignerMessage::BlockProposal(block_proposal) => {
+                StacksSignerMessage::BlockProposal(BlockProposalData {
+                    block: standardize_stacks_nakamoto_block(&block_proposal.block),
+                    burn_height: block_proposal.burn_height,
+                    reward_cycle: block_proposal.reward_cycle,
+                })
+            }
+            SignerMessage::BlockResponse(block_response) => match block_response {
+                BlockResponse::Accepted((block_hash, sig)) => StacksSignerMessage::BlockResponse(
+                    BlockResponseData::Accepted(BlockAcceptedResponse {
+                        block_hash: block_hash.to_hex(),
```

Comment:
I actually had this mislabeled initially in the js code. Technically it's actually `signer_signature_hash` (just like in the `BlockResponse::Rejected` case), however, it happens to be the same value as `block_hash`. Probably best to use the `signer_signature_hash` naming in case it doesn't match block_hash in epoch3.x.